src/index.html

```diff
@@ -11,7 +11,219 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-div-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="personalinfo">
+        <legend>
+          Personal information
+        </legend>
+
+        <label>
+          Surname:  
+          <input 
+            type="text"
+            class="surname" 
+            name="surname" 
+            required
+          >
+        </label>
+
+        <div class="one">  
+          <label>
+            Name:  
+            <input 
+              type="text"
+              class="name"
+              name="name" 
+              required
+            >      
+          </label>
+        </div>
+        
+        <div class="one">
+          <label>
+            How old are You?
+            <input 
+              type="number"
+              class="howold" 
+              name="howold" 
+              value="12" 
+              min="1" 
+              max="100"
+            >
+          </label>
+        </div>
+
+        <div class="one">
+          <label>
+            Full date of birth:
+            <input 
+              type="date"
+              class="date" 
+              name="date"
+            >
+          </label>
+        </div>
+
+        <div class="one">
+          <label>
+            I accept the term of the agreement
+            <input 
+              type="checkbox"
+              class="agree" 
+              name="agree" 
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+    
+      <fieldset class="personalinfo">
+        <legend>
+          Registration
+        </legend>
+
+        <label>
+          E-mail:
+          <input 
+            type="email"
+            class="email" 
+            name="email" 
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <div class="one">
+          <label>
+            Password:
+            <input type="password"
+            class="password" 
+            name="password" 
+            required
+            minlength="5"
+            maxlength="15"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="personalinfo">
+        <legend>
+          An interesting fact about you!
+        </legend>
+      
+        <label>
```

Comment:
not really good idea to use a common label here because when you click on the question, yes became selected